public/locales/en/peers.json

```diff
@@ -14,6 +15,7 @@
   "insertPeerAddress": "Insert the peer address you want to connect to.",
   "add": "Add",
   "example": "Example:",
+  "plusPeers": "Plus {number} more peers", 
```

Comment:
Would this look better or worse? (feel free to ignore)

```suggestion
  "plusPeers": "+ {number} more peers", 
```

Reply:
Don't really have a preference, just went with @jessicaschilling suggestion at the time 😄

Reply:
@lidel - I was waffling between "plus" and "+" in the first place, so if you'd prefer "+", let's go with it 😊